game/scripts/npc/abilities/windrunner_powershot.txt

```diff
@@ -39,7 +39,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_reduction"                                "20"
+        "damage_reduction"                                "0" //OAA
```

Comment:
I am not a fan of removing parts of spells that make those spells unique. I would scale it: 20/15/10/5/0/0.

Reply:
I have made the damage reduction 10% at all levels. There is basically no point in making this change if its 20% at level 1. Tbh if the damage penatly is 0% is it a "better" version of KOTL illuminate without aghs for early game farming.